repo-issues/lib/data-source.js

```diff
@@ -0,0 +1,97 @@
+const http = require('request-promise-native')
+const moment = require('moment')
+const octokit = require('@octokit/rest')()
+
+const show_details_threshold = 5
+const organisation = 'novoda'
+const languageIconMap = {
+    "html": "html5",
+    "groovy": "gradle",
+    "css": "css3",
+    "objective-c" : "c"
+}
+
+let showIssues = false
+let showPRs = true
+
+const generateViewState = (configuration) => {
+    showIssues = configuration.showIssues
+    showPRs = configuration.showPRs
+
+    octokit.authenticate({type: 'oauth', token: `${configuration.token}`})
+    
+    return octokit.repos.getForOrg({ org: organisation, type: 'all', per_page: 9999 })
```

Comment:
This was a workaround to avoid dealing with pages on the response. But turns out it doesn't work as expected (it only return 100 elements max)
❌ Refactoring after lunch to deal with pages properly